versioned_docs/version-V2/use-cases/private-voting.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 2
+---
+
+## Private voting overview
+
```

Comment:
@cedoor Still in draft, but I would appreciate your feedback on this `private-voting` page to make sure it's accurate.